### ISSUE TITLE
Include restaking and winddown in staker status output

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#1613]

**Comment by cygnusv** [comment]
What do you mean by staker status output? If you mean `nucypher status stakers`, both are shown:
```
0x2160DCf3EAE12e21DCC1b7294859d80Ba2065589  Nickname:  Crimson King Ivory Alembic ♚  ⚗
                                            Owned:     1531207.91 NU  (Staked: 1531207.91 NU)
                                            Re-staking: Yes  (Unlocked)
                                            Winding down: No
                                            Activity:  Next period confirmed (#18284)
                                            Worker:    0xbb0493Bda0D6B1CEA25972c28E9Bc26dc03fB910
```

**Comment by KPrasch** [comment]
Sorry for the spare description...

There are a few other output sinks: 

- [ ] Interactive Ursula Status
- [ ] `nucypher stake list`
- [ ] Ursula status webpage

**Comment by KPrasch** [comment]
Closed by #2824 and #2867
